content/en/docs/concepts/glossary.md

```diff
@@ -9,189 +9,369 @@ others. This page captures terminology used in the project and what it means.
+
+See [`Instrumented Library`](#instrumented-libraryspec-instrumented-lib).
+
+### **Client-side App**
+
+A component of an [`Application`](#application) that is not running inside a
+private infrastructure and is typically used directly by end-users. Examples of
+client-side apps are browser apps, mobile apps, and apps running on IoT devices.
+
+### **[Collector][]**
+
+A vendor-agnostic implementation on how to receive, process, and export
+telemetry data. A single binary that can be deployed as an agent or gateway.
+
+Also known as the OpenTelemetry Collector
+
+### **Contrib**
+
+Several
+[`Instrumentation Libraries`](#instrumentation-libraryspec-instrumentation-lib)
+and the [`Collector`](#collector) offer a set of core capabilities as well as a
+dedicated contrib repository for non-core capabilities including vendor
+`Exporters`.
+
+### **[Context Propagation][]**
+
+Allows all [`Data Sources`](#data-source) to share an underlying context
+mechanism for storing state and accessing data across the lifespan of a
+[`Transaction`](#transaction).
+
+### **[DAG][]**
+
+Directed Acyclic Graph.
+
+### **[Data Source][]**
+
+One of [`Traces`](#trace), [`Metrics`](#metric) or [`Logs`](#log).
+
+### **Dimension**
+
+See [`Label`](#label).
+
+### **[Distributed Tracing][]**
+
+Tracks the progression of a single [`Request`](#request), called a
+[`Trace`](#trace), as it is handled by [`Services`](#service) that make up an
+[`Application`](#application). A [`Distributed Trace`](#distributed-tracing)
+transverses process, network and security boundaries.
+
+### **Event**
+
+Something that happened where representation depends on the
+[`Data Source`](#data-source). For example, [`Spans`](#span).
+
+### **Exporter**
+
+Provides functionality to emit telemetry to consumers. Used by
+[`Instrumentation Libraries`][spec-exporter-lib] and the
+[`Collector`](/docs/collector/configuration#basics). Exporters can be push- or
+pull-based.
+
+### **[Field][]**
+
+Name/value pairs added to [`Log Records`](#log-record) (similar to
+[`Attributes`](#attribute) for [`Spans`](#span) and [`Labels`](#label) for
+[`Metrics`](#metric)).
+
+### **[gRPC](https://grpc.io)**
+
+A high-performance, open source universal [`RPC`](#rpc) framework.
+
+### **[HTTP][]**
+
+Hypertext Transfer Protocol.
+
+### **[Instrumented Library][spec-instrumented-lib]**
+
+Denotes the [`Library`](#library) for which the telemetry signals
+([`Traces`](#trace), [`Metrics`](#metric), [`Logs`](#log)) are gathered.
+
+### **[Instrumentation Library][spec-instrumentation-lib]**
+
+Denotes the [`Library`](#library) that provides the instrumentation for a given
+[`Instrumented Library`](#instrumented-libraryspec-instrumented-lib).
+[`Instrumented Library`](#instrumented-libraryspec-instrumented-lib) and
+[`Instrumentation Library`](#instrumentation-libraryspec-instrumentation-lib)
+may be the same [`Library`](#library) if it has built-in OpenTelemetry
+instrumentation.
+
+### **[JSON][]**
```

Comment:
Is this missing a link to the official site of JSON?

Reply:
Nope - I just checked the link and it works. The ref is at the very bottom of the page.